underpants.go

```diff
@@ -247,6 +267,22 @@ func urlFor(scheme, host string, r *http.Request) *url.URL {
   return &u
 }
 
+func userMemberOf(c *conf, u *user, groups []string) bool {
+  for _, group := range groups {
+    if group == "*" {
+      return true
+    }
+
+    for _, allowedUser := range c.Groups[group] {
```

Comment:
Is linear search fine for your use cases? I can change this to a constant time lookup if we need it in the future.